tests/auth/test_controller.py

```diff
@@ -267,4 +269,123 @@ def tearDown(self):
         with self.app.app_context():
             db.session.remove()
             db.drop_all()
-            db.engine.dispose()
\ No newline at end of file
+            db.engine.dispose()
+
+
+class AuthorizeUploadTestCase(unittest.TestCase):
```

Comment:
I'd assumed this change was more a refactor. If we are adding this test case where are the tests we are deleting?